stable/redis/templates/_helpers.tpl

```diff
@@ -103,6 +103,7 @@ readinessProbe:
     - sh
     - -c
     - /health/ping_local_and_master.sh
+    - {{ $readinessProbe.timeoutSeconds | default .Values.master.readinessProbe.timeoutSeconds }}
```

Comment:
The tests are failing, could you adapt the command to use so it looks like the one below:

```
         - bash
         - -ec
         - |
            /health/ping_local_and_master.sh
            {{ $livenessProbe.timeoutSeconds | default .Values.master.livenessProbe.timeoutSeconds }}
```

I think that should fix the tests.

Reply:
I changed it the same way I changed the statefulset, let me know if you prefer both be the way you suggested instead.

Reply:
It's okay like you did. Thanks!